pom.xml

```diff
@@ -62,9 +62,8 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <revision>2.22</revision>
-        <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.277.1</jenkins.version>
+        <changelist>999999-SNAPSHOT</changelist>
+        <jenkins.version>2.304</jenkins.version>
```

Comment:
Any particular reason? Amended in #119.

Reply:
No there was no point. I can only assume it was copy and paste error